subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -112,7 +112,7 @@ abstract class AbstractSmokeTest extends Specification {
         })
 
         // https://plugins.gradle.org/plugin/org.gretty
-        static gretty = "3.0.6"
+        static gretty = "4.0.0"
```

Comment:
Should we test both versions? I don't know if their is still some development on the 3.x line or if that stopped now.

Reply:
We can test 3.0.x with Java8 and for Java11 we use Gretty 4. I saw there was small update 28 days ago (3.0.7), although just some groovy dependencies fix

Reply:
I think that would be good.